docs/docs/datasources.md

```diff
@@ -48,12 +48,12 @@ spec:
     basicAuth: true
     url: http://prometheus-service:9090
     isDefault: true
-    user: ${PROMETHEUS_USERNAME}
+    basicAuthUser: ${PROMETHEUS_USERNAME}
     jsonData:
       "tlsSkipVerify": true
       "timeInterval": "5s"
     secureJsonData:
-      "password": ${PROMETHEUS_PASSWORD} # Notice the brakes around PROMETHEUS_PASSWORD
+      "basicAuthPassword": ${PROMETHEUS_PASSWORD} # Notice the brakes around PROMETHEUS_PASSWORD
```

Comment:
would you mind changing `brakes` to `braces` while you're at it :wink:

Reply:
Done